package.json

```diff
@@ -21,18 +21,19 @@
     "@angular/common": "~7.2.14",
     "@angular/compiler": "~7.2.14",
     "@angular/core": "~7.2.14",
-    "@angular/fire": "^5.1.2",
+    "@angular/fire": "^5.1.3",
     "@angular/flex-layout": "^7.0.0-beta.24",
     "@angular/forms": "~7.2.14",
     "@angular/material": "^7.3.7",
     "@angular/platform-browser": "~7.2.14",
     "@angular/platform-browser-dynamic": "~7.2.14",
     "@angular/router": "~7.2.14",
-    "@firebase/app": "^0.3.16",
-    "@firebase/app-types": "^0.3.9",
-    "angulartics2": "^7.5.1",
+    "@firebase/app": "^0.3.17",
+    "@firebase/app-types": "^0.3.10",
+    "@firebase/performance-types": "0.0.1",
+    "angulartics2": "^7.5.2",
     "core-js": "^2.5.4",
-    "firebase": "^5.10.0",
+    "firebase": "~5.10.0",
```

Comment:
This is required since 5.11.0 pulls in a different version of `@firebase/performance' and doesn't seem to be compatible with the EAP.